sosw/components/helpers.py

```diff
@@ -887,3 +890,33 @@ def to_bool(val):
         elif val.lower() in ['false', '0']:
             return False
     raise Exception(f"Can't convert unexpected value to bool: {val}, type: {type(val)}")
+
+
+def get_message_dict_from_sns_event(event):
+    """
+    Extract SNS event message and returns it as dict
+
+    :param dict event: SNS Event (payload)
```

Comment:
I would say
`:param dict event: Lambda SNS event (payload)`
`:return: The SNS message, converted to dict`